### PR TITLE
Adjust prettify colors

### PR DESCRIPTION
**Pull request by norkans7**
Fixes https://github.com/rapidpro/rapidpro/issues/1363

### CODE REVIEW

**Comment by norkans7** [comment]
Before
Channel Event 2020-10-27 15-29-49



After
Channel Event 2020-10-27 15-30-02

File: static/rest_framework/css/prettify.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.com { color: #93a1a1; }
+.lit { color: #195f91; }
+.pun, .opn, .clo { color: #93a1a1; }
+.fun { color: #bdb76b;}
+.str, .atv { color: #4787ea;}
+.kwd, .prettyprint .tag { color: #1e347b; }
```
Comment (norkans7) [inline static/rest_framework/css/prettify.css:6]: The only changed lines

```
.fun { color: #bdb76b;}
.str, .atv { color: #4787ea;}
```

Comment (norkans7) [reply static/rest_framework/css/prettify.css:6]: from 
```
.fun { color: #dc322f; }
.str, .atv { color: #D14; }
```

**Review by rowanseymour — APPROVED** [review]
Looks good. Long term seems like this would be a good component - means those css styles don't leak elsewhere

**Pull request merged**